uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/Entitlement.php

```diff
@@ -1,249 +0,0 @@
-<?php
```

Comment:
Yes. It was replaced by the LeavePeriodEntitlement entity: https://github.com/civicrm/civihr/pull/1078/files/475ccfbec867d6df85cb9135720362d10e91b5a4#diff-c9b255fb4e8e91bee16a7b2970371439R70